### PR TITLE
Portfolio additions1

### PR DESCRIPTION
**Pull request by abelfeleke**
Added some basic features to the website, nothing crazy.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,19 +1,71 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <!-- Style -->
+    <style> 
+
+        /* Background Color */
+        body {
```
Comment (thechungster) [inline portfolio/src/main/webapp/index.html:8]: You can add these styles to the `portfolio/src/main/webapp/style.css` file so the css and html are split into separate files

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -7,3 +7,8 @@
 #greeting-container {
   margin-top: 20px;
 }
+
```
Comment (thechungster) [inline portfolio/src/main/webapp/style.css:10]: This doesn't seem needed

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -17,7 +17,8 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['I spend a lot of time on YouTube.', 'I am watching 3 different anime shows right now.',
```
Comment (thechungster) [inline portfolio/src/main/webapp/script.js:20]: I think I've these changes before, were you able to merge the previous pull request?

**Review by thechungster — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (abelfeleke) [reply portfolio/src/main/webapp/script.js:20]: Just did it, I forgot!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (abelfeleke) [reply portfolio/src/main/webapp/index.html:8]: Once I copy my style code to the css file, the style doesn't show up on my server for some reason. I just asked my team for help.

Comment (abelfeleke) [reply portfolio/src/main/webapp/index.html:8]: Figured it out,

Comment (abelfeleke) [reply portfolio/src/main/webapp/index.html:8]: .*

**Pull request closed without merging**